pinot-tools/src/main/resources/examples/batch/airlineStats/airlineStats_offline_table_config.json

```diff
@@ -40,5 +42,21 @@
         "transformFunction": "fromEpochDays(DaysSinceEpoch)"
       }
     ]
-  }
+  },
+  "tierConfigs": [
+    {
```

Comment:
wont setting this config automatically make all quickstarts do tier movements? as SegmentRelocator is on by default?

Reply:
Good question. With tierConfigs set in tableConfig, segments will have target tier set accordingly, but nothing happens upon getting new tiers. Because those two configs are needed to turn on the tier migration across multi-dirs. 
```
    properties.put("pinot.server.instance.segment.directory.loader", "tierBased"); // this segment loaders knows how to migrate across data paths on servers
    properties.put("controller.segmentRelocator.enableLocalTierMigration", "true"); // this tells servers to migrate data paths
```